lib/LedgerSMB/Scripts/setup.pm

```diff
@@ -1073,7 +1073,8 @@ sub save_user {
     use LedgerSMB::PGDate;
 
     _init_db($request);
-    $request->{dbh}->{AutoCommit} = 0; #Enable transactions, must manage.
+    #We enable transactions, we will have to manage commits and rollbacks ourselves.
```

Comment:
From https://ledgersmb.org/content/coding-guidelines (emphasis mine):
> comments should describe things which are __*outside the code*__ -- why's and wherefore's, promises to developers, reasoning, etc.

Can you explain what's *outside the code* about your comment? I think 'we enable transactions' is an exact description *of* the code (so, it's *in* the code). The immediate consequence of it is then that we need to - of course - finish the transactions.

A comment which explains what's outside this direct code is:

```
# The request dispatcher will commit the transaction for us (or rollback in case of error)
```

However, it seems we don't depend on that behaviour here(?) given the rollbacks below, so, it would be a misleading comment in this exact case.